src/Unzer.php

```diff
@@ -63,7 +63,7 @@ class Unzer implements UnzerParentInterface, PaymentServiceInterface, ResourceSe
     public const BASE_URL = 'api.unzer.com';
     public const API_VERSION = 'v1';
     public const SDK_TYPE = 'UnzerPHP';
-    public const SDK_VERSION = '1.1.0.0';
+    public const SDK_VERSION = '1.1.0.1';
```

Comment:
It is a feature no bugfix.
```suggestion
    public const SDK_VERSION = '1.1.1.0';
```